src/core/events/CoreEvents.js

```diff
@@ -58,6 +58,7 @@ class CoreEvents extends EventsBase {
         this.LOADING_PROGRESS = 'loadingProgress';
         this.LOADING_DATA_PROGRESS = 'loadingDataProgress';
         this.LOADING_ABANDONED = 'loadingAborted';
+        this.MANIFEST_LOADING_STARTED = 'manifestLoadingStarted';
```

Comment:
You are using this event in your application? Then please add it to `MediaPlayerEvents` , `CoreEvents` are internal events that should not be needed at the player level.